.github/workflows/ci.yml

```
@@ -84,7 +84,7 @@ jobs:
           PYTHON: ""
           SBP_TEST: ${{ matrix.sbp_test }}
       - uses: julia-actions/julia-processcoverage@v1
-      - uses: codecov/codecov-action@v4
+      - uses: codecov/codecov-action@v5
         with:
           file: ./lcov.info
```

```suggestion
          files: ./lcov.info
```
The `file` keyword has been deprecated in favor of `files`, cf. https://github.com/codecov/codecov-action?tab=readme-ov-file#migration-guide.

---

Thanks